spec/pages/posts_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe 'Page' do
+  let(:fullscreen_page_id) { 'FullscreenInc' }
+
+  describe '#posts' do
+    let(:page) { Funky::Page.find(page_id) }
+    let(:posts) { page.posts }
+
+    context 'given an existing page ID was passed' do
+      let(:page_id) { fullscreen_page_id }
+
+      specify 'returns a list of posts' do
+        post = posts.first
+
+        expect(posts).to all be_a Funky::Post
+        expect(post.type).to be_a String
+        expect(post.count_shares).to be_a Integer
```

Comment:
@kangkyu Either with 0 or with nil, we need a test with a post that does not have shares.